libs/service-portal/settings/access-control/src/screens/GrantAccess/GrantAccess.tsx

```diff
@@ -237,9 +239,12 @@ function GrantAccess() {
               ) : null}
             </div>
           </GridColumn>
-          <GridColumn span={['12/12', '12/12', '4/12']}>
+          <GridColumn
+            span={['12/12', '12/12', '4/12']}
+            className={styles.button}
```

Comment:
Doesn't this look weird if the input renders a validation issue, below the input, making the button misalign?